docs/mp/config/01_introduction.adoc

```diff
@@ -133,52 +136,12 @@ client_secret=${CLEAR=known_password}
 ----
 
 * *Meta Configuration* +
-You can configure the Config using Helidon MP Config meta configuration feature.
-This is a Helidon specific feature available since version 2.3.0.
-
-When used, the {spec-name} uses configuration sources and flags configured in the meta configuration file.
+You can configure the Config using Helidon MP Config meta configuration feature. The meta-config allows configuration of config sources and other
+configuration options, including addition of discovered sources and converters.
 
-The meta-config allows configuration of config sources and other configuration options,
-including addition of discovered sources and converters.
+This is a Helidon specific feature available since version 2.3.0. See <<mp/config/02_MP_config_sources.adoc,Microprofile Config Sources>> for detailed information.
 
-If a file named `mp-meta-config.yaml`, or `mp-meta-config.properties` is in the current directory or
-on the classpath, and there is no explicit setup of configuration in the code, the configuration will
-be loaded from the `meta-config` file.
-The location of the file can be overridden using system property `io.helidon.config.mp.meta-config`,
-    or environment variable `HELIDON_MP_META_CONFIG`
-
-[source,yaml]
-.Example of a YAML meta configuration file:
-----
-add-discovered-sources: true <1>
-add-discovered-converters: false <2>
-add-default-sources: false <3>
-
-sources:
-  - type: "environment-variables" <4>
-  - type: "system-properties" <5>
-  - type: "properties" <6>
-    path: "/conf/prod.properties" <7>
-    ordinal: 50 <8>
-    optional: true <9>
-  - type: "yaml"  <10>
-    classpath: "META-INF/database.yaml" <11>
-
-----
-
-<1> If configured to `true`, config sources discovered through service loader will be added
-<2> If configured to `true`, converters discovered through service loader will be added
-<3> If configured to `true`, default config sources (system properties, environment variables, and `META-INF/microprofile-config.properties) will be added
-<4> Loads the environment variables config source.
-<5> Loads the system properties config source.
-<6> Loads a properties file
-<7> Location of the file: `/conf/prod.properties` on the file system
-<8> Custom ordinal, if not defined, the value defined in the file, or default value is used
-<9> The file is optional (if not optional and no file is found, the bootstrap fails)
-<10> Loads a YAML file
-<11> Location of the file: `META-INF/database.yaml` on the classpath
-
-For backward compatibility, we will support usage of Helidon SE meta-configuration until version 3.0.0. Using this approach causes behavior that is not compatible with {spec-name} specification.
+NOTE: For backward compatibility, we will support usage of Helidon SE meta-configuration until version 3.0.0. Using this approach causes behavior that is not compatible with {spec-name} specification.
```

Comment:
You have removed the MP Meta configuration. That should not have been removed - this is a Helidon MP specific feature (and is not related to SE meta configuration).

Reply:
I have just moved it from here to second chapter which is more of example and linked in from this section.